packages/terra-collapsible-menu-view/src/CollapsibleMenuViewItemGroup.jsx

```diff
@@ -70,18 +70,22 @@ class CollapsibleMenuViewItemGroup extends React.Component {
     const { isCollapsibleMenuItem } = this.context;
 
     if (isCollapsibleMenuItem && selectedKeys.length) {
-      return (
-        <Menu.ItemGroup {...customProps} onChange={this.handleOnChange}>
-          {children}
-        </Menu.ItemGroup>
-      );
-    } if (isCollapsibleMenuItem) {
-      return (
-        <List {...customProps} onChange={this.handleOnChange}>
-          {children}
-        </List>
-      );
-    }
+          return (
+            <li>
+              <Menu.ItemGroup {...customProps} onChange={this.handleOnChange}>
+                {children}
+              </Menu.ItemGroup>
+            </li>
+          );
+        } if (isCollapsibleMenuItem) {
+          return (
+            <li>
+              <List {...customProps} onChange={this.handleOnChange}>
```

Comment:
Hmmm doesn't this throw an accessibility error or screen reader error by having a list nested in a list item??

Reply:
I believe these get nested in a UL. Before this change we had the following structure:

```
ul
  ul 
    li
```

This updates wraps the nested `ul` in a `li`.

```diff
ul
+  li
    ul 
      li
```